software/acpid.txt

```diff
@@ -0,0 +1,123 @@
+ACPID
+________________________________________________________________________________
+
+
+Prerequisites
+________________________________________________________________________________
+
+Make sure that ACPI-related kernel options are enabled. [0]
+
+
+Using busybox acpid
+________________________________________________________________________________
+
+Busybox provides a very limited acpid program. The only event that it works
+well for is closing the lid. For some users this is enough. To run
+commands when the lid is closed, create an executable script at
+/etc/acpi/LID/00000080. The example below will suspend your computer to RAM.
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   #!/bin/sh                                                                  |
+|                                                                              |
+|   printf mem > /sys/power/state                                              |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+The rest of this article uses the more robust acpid program from the acpid package in the community repository. Install it as usual:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   kiss build acpid                                                           |
+|   kiss install acpid                                                         |
```

Comment:
See: k1ss.org/wiki/help/page-template for how command installation should be formatted.